superset-frontend/plugins/plugin-chart-echarts/src/utils/series.ts

```diff
@@ -88,9 +88,6 @@ export function formatSeriesName(
   if (name === undefined || name === null) {
     return NULL_STRING;
   }
-  if (typeof name === 'number') {
```

Comment:
Sorry for this stupid question: just move the location? Did the code change?

Reply:
The order of these affects the result - temporal columns are almost always epochs (`number`), and therefore the number formatter was always used on them. Now the "original" type of the column is first checked (based on `is_dttm` coming from the backend), so temporal numerical names will now be properly identified.

Reply:
Yep, just like Ville said :)

Reply:
thanks for the explanation.